os/tools/mkbinheader.py

```diff
@@ -116,206 +170,189 @@ def get_static_ram_size(bin_type):
+    SCHED_PRIORITY_MIN = 1
+    SCHED_PRIORITY_MAX = 255
 
-if int(main_stack_size) >= int(dynamic_ram_size) :
-    print("Error : Dynamic ram size should be bigger than Main stack size.")
-    print("Dynamic ram size : %d, Main stack size : %d" %(int(dynamic_ram_size), int(main_stack_size)))
-    sys.exit(1)
+    # BM priority MAX/MIN
+    BM_PRIORITY_MAX = 205
+    BM_PRIORITY_MIN = 200
 
-config_path = os.getenv('TOPDIR') + '/.config'
-priority = get_config_value(config_path, "CONFIG_BM_PRIORITY_MAX=")
-if priority > 0 :
-	BM_PRIORITY_MAX = priority
-priority = get_config_value(config_path, "CONFIG_BM_PRIORITY_MIN=")
-if priority > 0 :
-	BM_PRIORITY_MIN = priority
-
-with open(file_path, 'rb') as fp:
-    # binary data copy to 'data'
-    data = fp.read()
-    file_size = fp.tell()
-    fp.close()
-
-    if binary_type == 'bin' or binary_type == 'BIN' :
-        bin_type = BIN
-    elif binary_type == 'elf' or binary_type == 'ELF' :
-        bin_type = ELF
-    else : # Not supported.
-        bin_type = 0
-        print("Error : Not supported Binary Type")
+    if int(main_stack_size) >= int(dynamic_ram_size) :
+        print("Error : Dynamic ram size should be bigger than Main stack size.")
+        print("Dynamic ram size : %d, Main stack size : %d" %(int(dynamic_ram_size), int(main_stack_size)))
         sys.exit(1)
 
-    # Calculate RAM size
-    # Dynamic RAM size : user input of argv[6]
-    # Static RAM size : Extract from size command in linux(ONLY for elf)
-    if bin_type == BIN :
-        os.system('size ' + elf_path_for_bin_type + ' > ' + STATIC_RAM_ESTIMATION)
-    elif bin_type == ELF :
-        os.system('readelf -S ' + file_path + ' > ' + STATIC_RAM_ESTIMATION)
-    else : #Not supported.
-        print("Error : Not supported Binary Type")
-        sys.exit(1)
-
-    main_priority = int(main_priority)
-    if (main_priority < SCHED_PRIORITY_MIN) or (main_priority > SCHED_PRIORITY_MAX) or (BM_PRIORITY_MIN <= main_priority and main_priority <= BM_PRIORITY_MAX) :
-        print("Error : This binary priority ", main_priority, " is not valid")
-        sys.exit(1)
+    config_path = CONFIG_DIR + '/.config'
+    priority = get_config_value(config_path, "CONFIG_BM_PRIORITY_MAX=")
+    if priority > 0 :
+        BM_PRIORITY_MAX = priority
+    priority = get_config_value(config_path, "CONFIG_BM_PRIORITY_MIN=")
+    if priority > 0 :
+        BM_PRIORITY_MIN = priority
+
+    with open(file_path, 'rb') as fp:
+        # binary data copy to 'data'
+        data = fp.read()
+        file_size = fp.tell()
+        fp.close()
+
+        if binary_format == 'elf' or binary_format == 'ELF' :
+            bin_type = ELF
+        else : # Not supported.
+            bin_type = 0
+            print("Error : Not supported Binary Type")
+            sys.exit(1)
+
+        # Calculate RAM size
+        # Dynamic RAM size : user input of argv[6]
+        # Static RAM size : Extract from size command in linux(ONLY for elf)
```

Comment:
There is no usage for using size command in linux anymore. Please remove this explanation.